tutor/config.py

```diff
@@ -185,15 +187,13 @@ def convert_json2yml(root: str) -> None:
         return
     if os.path.exists(config_path(root)):
         raise exceptions.TutorError(
-            "Both config.json and config.yml exist in {}: only one of these files must exist to continue".format(
-                root
-            )
+            f"Both config.json and {_CONFIG_FILE} exist in {root}: only one of these files must exist to continue"
```

Comment:
f-string formatting is not supported in Python 3.5, thus we should avoid it. Same for the change below and in tests/test.py.

Reply:
OK I just learned understand that Python 3.5 had reached end of life a year ago and that Tutor was not compatible anymore because we depend on astroid 2.8.3 :sweat_smile: Nonetheless, let's not add f-strings for now, and we'll add them after we explicitely drop Python 3.5 compatibility from setup.py.

Reply:
Here is the PR that drops Python 3.5 compatibility (in the nightly branch): https://github.com/overhangio/tutor/pull/542

Reply:
@regisb since you already [bump up the minimum python requirement to 3.6](https://github.com/overhangio/tutor/pull/542/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R55) now, do you still want me to separate all f-string formatting as a separate PR in the future?

Reply:
Well, yes because we only dropped the python 3.5 compatibility in the nightly branch, while your PR targets the master branch.

Reply:
All f-strings are replaced with format strings for this PR.